apps/crm/crm/src/app/crm/pages/organizations/organizations.component.html

```diff
@@ -19,27 +19,35 @@
     </div>
   </article>
 
-  <bf-table clickable [source]="orgs" pagination="50" useFilter (rowClick)="goToEdit($event)">
+  <bf-table [source]="orgs" pagination="50" useFilter>
     <ng-template colRef="id" let-id sort>{{ id }}</ng-template>
     <ng-template colRef="status" let-status sort>
       <bf-tag>{{ status }}</bf-tag>
     </ng-template>
-    <ng-template colRef="denomination.full" label="Name" sort let-fullName>
+    <ng-template colRef="denomination.full" label="Organization Name" sort let-fullName>
       {{ fullName }}
     </ng-template>
     <ng-template colRef="addresses.main.country" defaultSort label="Country" sort let-country>
       {{ country ?? '--' | titlecase }}
     </ng-template>
+    <ng-template colRef label="Status" sort let-org>
+      {{ org.status ?? '--' | toLabel: 'organizationStatus' }}
```

Comment:
Do you  need `?? --` ? I think that every orgs have status
```suggestion
      {{ org.status ?? '--' | toLabel: 'organizationStatus' }}
```